scripts/vep/run_vep.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+
+"""
+Run VEP on the hail mt
+"""
+
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+
+@click.command()
+@click.option('--mt', required=True, help='Hail matrix table to run VEP on')
+def main(mt: str):
+    """
+    Run vep using main.py wrapper
+    """
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(mt)
+    # filter to biallelic loci only
+    mt = mt.filter_rows(hl.len(mt.alleles) == 2)
+    mt = mt.filter_rows(mt.alleles[1] != '*')
```

Comment:
Weren't we planning on skipping this filter? And I think there were a few other suggestions in the other PR.

Reply:
I don't think VEP works with starred alleles: https://discuss.hail.is/t/vep-output-variant-not-found-in-original-variants/1148

Reply:
Ah, thanks. Could you add a comment explaining that, please?